docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:

```

Comment:
This file isn't mentioned in your README...

But also I don't expect it to work because e.g. the e2e tests need real credentials and you are trying to run all the tests with just .env.test

It would be more useful for the docker-compose file to run the dev image.
It can also take care of the build step if you use 
```
    build: 
      context: .
      target: development
```
 instead of `image: your-image`
 
 Just make sure to run `docker-compose up --build` to ensure the latest Dockerfile is used
 
 The test run does not need ports to be published, nor a volume mounted